docs/modders/Campaign_Format.md

```diff
@@ -182,6 +182,8 @@ Predefined campaign regions are located in file `campaign_regions.json`
 
 ```js
 {
+    "background": "own_region_background.bmp",
+		"suffix": ["Enabled", "Selected", "Conquered"],
     "prefix": "G3",
     "color_suffix_length": 1,
```

Comment:
I know this is not yours code, but can we change it to `colorSuffixLength` before we have campaigns we need to support?
 We always use camelCase in json's